app/js/controllers.js

```diff
@@ -1,5 +1,7 @@
-var conceptDirControllers = angular.module('conceptDirControllers', []);
+var conceptDictControllers = angular.module('conceptDictControllers', []);
+
+conceptDictControllers.controller('ClassesListCtrl', ['$scope', 'Classes', function($scope, Classes) {
+	  $scope.classObj = Classes.query();	 
```

Comment:
Change to classes and make the query return an array instead of an object with the results array.